src/asl/krislet.asl

```diff
@@ -9,10 +9,10 @@
 +!start : not ballVisible(BallDist, BallDir) & not ballClose <-
 	if (ballRightOfPlayer){
 		turn(30);
-		.wait(200);
+		//.wait(200);
```

Comment:
@LukeANewton We don't need to call internal .wait(200) in asl which delays the action socket sending to server.